symfony/plugins/orangehrmRESTPlugin/config/orangehrmRESTPluginConfiguration.class.php

```diff
@@ -15,7 +15,7 @@ public function initialize() {
         $enabledModules = sfConfig::get('sf_enabled_modules');  
         if (is_array($enabledModules)) {  
             sfConfig::set('sf_enabled_modules',
-                array_merge(sfConfig::get('sf_enabled_modules'), array('baseapi','apiv1pim','apiv1leave','apiv1admin','apiv1time','apiv1attendance','apiv1performance','apiv1integration'))
+                array_merge(sfConfig::get('sf_enabled_modules'), array('baseapi','apiv1pim','apiv1leave','apiv1admin','apiv1time','apiv1attendance','apiv1performance','apiv1integration','apiv1mobile'))
```

Comment:
Why myInfo part of mobile plugin? is it not match with the PIM?

Reply:
Done